src/lib/core/components/computations/RunComputeButton.tsx

```diff
@@ -53,14 +53,26 @@ const colorMap: Record<JobStatus, string> = {
   failed: 'red',
 };
 
+// Replace internal job status with user-friendly status messages
+const jobStatusDisplay: Record<JobStatus, string> = {
+  'no-such-job': 'Not started.',
+  requesting: 'Requesting.',
+  queued: 'Queued.',
+  'in-progress':
+    'In progress. You may return later to use results in the visualization.',
+  complete: 'Complete, results saved in the system.',
```

Comment:
I'd probably be consistent here and make this two sentences: `'Complete. Results saved in the system.'`

Just a thought. Not necessary at this stage though.

Reply:
good catch!